backend/scripts/load_test.py

```
@@ -0,0 +1,512 @@
+            # Test login endpoint
+            tasks = []
+            for i in range(num_users):
+                user_creds = TEST_USERS[i % len(TEST_USERS)]
+                task = login_user(session, user_creds["email"], user_creds["password"])
+                tasks.append(task)
+            
+            start = time.time()
+            login_results = await asyncio.gather(*tasks, return_exceptions=True)
+            elapsed = time.time() - start
+            
+            for i, result in enumerate(login_results):
```

Ruff (0.14.4)

Loop control variable `i` not used within loop body

(B007)